cypress/integration/FileInputFieldWithList/common/index.js

```diff
@@ -0,0 +1,26 @@
+import { Given, Then } from 'cypress-cucumber-preprocessor/steps'
+
+Given('a FileInputFieldWithList with multiple files is rendered', () => {
+    cy.visitStory('FileInputFieldWithList', 'Multiple files - with files')
+
+    cy.window().then(win => {
+        // The property has to be present to allow cy.stub
+        win.onChange = () => {}
+        cy.stub(win, 'onChange')
+    })
+})
+
+Then('the onChange handler is called', () => {
+    cy.window().then(win => {
+        const calls = win.onChange.getCalls()
+        expect(calls).to.have.lengthOf(1)
+
+        const callArgs = calls[0].args
+        expect(callArgs).to.have.lengthOf(2)
+
+        const payload = callArgs[0]
+        expect(Object.keys(payload)).to.include.members(['files', 'name'])
+
+        cy.wrap(payload).as('payload')
+    })
```

Comment:
I'm sure there's a "neat" cypress way for this as well

Reply:
This can be addressed in another PR. I have simply taken this from the FileInput code.